components/eamxx/src/diagnostics/tests/field_at_height_tests.cpp

```diff
@@ -128,22 +148,22 @@ TEST_CASE("field_at_height")
     print(" -> Testing with z_tgt coinciding with a z level\n");
     {
       print("    -> scalar midpoint field...............\n");
-      auto d = run_diag (s_mid,z_mid,loc);
+      auto d = run_diag (s_mid,z_mid,loc,"sealevel");
```

Comment:
Looks like you only test `"sealevel"`. Can we add some tests for the `"surface"` case?